packages/components/src/components/guild-navbar.tsx

```diff
@@ -52,3 +52,23 @@ export function GuildUnifiedLogo({
     </>
   );
 }
+
+/**
+ * GraphQL-related products live under the Hive platform brand, so we use a single logo for them.
+ * The rest gets The Guild / {Product} logo.
+ */
+export function getNavbarLogo(
+  logo: DocsThemeConfig['logo'],
+  websiteName: string,
+  description: string,
+) {
+  return logo === HivePlatformLogo ? (
```

Comment:
this condition will be not executed if logo are passed as ReactNode, e.g. `<HivePlatformLogo className="w-8" />`

let's relay on `websiteName` for now https://github.com/kamilkisiela/graphql-hive/blob/16a2f92ebf004096bb23c3b88f0fc9d4aba14c92/packages/web/docs/theme.config.tsx#L61

Reply:
Updated to use `websiteName === 'Hive'` and render `logo` when a function or ReactNode is passed.